libraries/api/test/logging_test.js

```diff
@@ -173,30 +179,32 @@ suite(testing.suiteName(), function() {
     });
     await request.get(url).set('Authorization', header);
 
-    assert.equal(helper.monitorManager.messages.length, 1);
-    delete helper.monitorManager.messages[0].Fields.duration;
-    delete helper.monitorManager.messages[0].Fields.expires;
-    assert.deepEqual(helper.monitorManager.messages[0], {
-      Type: 'monitor.apiMethod',
-      Severity: LEVELS.notice,
-      Fields: {
-        name: 'sometimesRequireNoScopes',
-        apiVersion: 'v1',
-        clientId: 'client-with-aa-bb-dd',
-        hasAuthed: true,
-        method: 'GET',
-        public: false,
-        query: {
-          private: "0",
+    await testing.poll(async () => {
+      assert.equal(helper.monitorManager.messages.length, 1);
+      delete helper.monitorManager.messages[0].Fields.duration;
+      delete helper.monitorManager.messages[0].Fields.expires;
+      assert.deepEqual(helper.monitorManager.messages[0], {
+        Type: 'monitor.apiMethod',
+        Severity: LEVELS.notice,
+        Fields: {
+          name: 'sometimesRequireNoScopes',
+          apiVersion: 'v1',
+          clientId: 'client-with-aa-bb-dd',
+          hasAuthed: true,
+          method: 'GET',
+          public: false,
```

Comment:
This test is failing but passes if we set `public` to `true`. I believe it should be `true` since there are no scopes required. @petemoore can you confirm?

Reply:
That's right - `public` implies that the rendered scope template for the current request has no scopes.